docs/development.rst

```diff
@@ -0,0 +1,47 @@
+Development
+===========
+
+Testing
+-------
+
+Make sure you have all the required modules before running the tests
+(modules that are only needed for tests are not included during
+installation)::
+
+
+    $ pip install -r requirements.txt
+
+
+Now run the unit tests and print out code coverage with `py.test`::
+
+    $ python -m pytest --cov=anonlinkclient
```

Comment:
We should just configure this in `setup.cfg`. See for example https://github.com/hardbyte/python-can/blob/develop/setup.cfg#L8